scripts/contentScript.js

```diff
@@ -666,7 +808,62 @@ const orchestrator = (tabUrl, websiteUrl, userId) => {
 				}
 			}
 		}
-	})
+
+		if (urlObj[2] === 'gitlab.com') {
+
+			if(urlObj.length === 4 && !urlObj[3]){
+				//TODO: need repo_owner from userID
+								
+
+			}
+
+			else if (urlObj.length > 4){	
+
+				const ownerName = urlObj[3];
+				const repoName = urlObj[4];
```

Comment:
I don't think this will always be the case. I suggest that you exactly specify the URLs where you want this behaviour.
My guess from this code is that you don't want `urlObj[3]` or `urlObj[4]` to be any of the keywords. I think you have referenced one of the other GitHub conditions where we checked if the urlObj[3] is not equal to "org".

LOGAF: 5/10

Reply:
No i believe it is correct. For a url like `https://gitlab.com/rhydberg/the-project-of-all-time` the urlObj turns out to be:
`0:"https:"
1: ""
2:"gitlab.com"
3:"rhydberg"
4:"the-project-of-all-time"`